<launch>
  <node name="uvc_camera" pkg="uvc_camera" type="camera_node"
        respawn="false" output="screen">
    <param name="camera_info_url" value="file://$(find roseus_tutorials)/launch/usb-camera.yaml"/>
    <param name="device" value="/dev/video0"/>
    <param name="width" value="640" />
    <param name="height" value="480" />
    <param name="frame_id" value="uvc_camera"/>
    <param name="frame_id" value="uvc_camera"/>
  </node>

  <node name="image_proc" pkg="image_proc" type="image_proc"
	respawn="true">
  </node>

  <node name="imagesift" pkg="imagesift" type="imagesift"
	respawn="false" output="screen" launch-prefix="nice -n 10">
    <remap from="image" to="image_rect_color" />
  </node>

  <node name="point_pose_extractor" pkg="jsk_perception" type="point_pose_extractor"
	respawn="false" output="screen"/>

</launch>
